tests/zfs-tests/include/libtest.shlib

```diff
@@ -1147,21 +1146,21 @@ function is_shared
 	fi
 
 	if is_linux; then
-		for mtpt in `$SHARE | $AWK '{print $1}'` ; do
+		for mtpt in `share | awk '{print $1}'` ; do
```

Comment:
AFAIK linux does not yet provide a single `share` binary to handle both NFS and SMB: every "share" and "unshare" call will have to be replaced with function wrappers.